lib/messaging/messagehandler.rb

```diff
@@ -22,7 +28,7 @@ class MessageHandler
 
   def initialize
     #@dbaccess = DbAccess.new
-    @dbaccess = $dbaccess
+    @dbaccess = $dbaccess || NullDatabase.new
```

Comment:
I was actually expecting something like farmtalk.rb, analogous to farmbot.rb, that would only start the mesh communication part for testing.